src/pages/references/authoring/editor/classes/ItemList.md

```diff
@@ -104,11 +107,11 @@ Iterates over all the items in this list. Mutations that occur mid-iteration are
 
 #### Inherited from
 
-[ReadOnlyItemList](ReadOnlyItemList.md).[iterator](ReadOnlyItemList.md#iterator)
+[RestrictedItemList](RestrictedItemList.md).[[iterator]](RestrictedItemList.md#[iterator])
```

Comment:
It's weird that this "Inherited from" generates a link to the superclass, but none of the other ones do. Not sure if that's another Typedoc bug, or what 🤔

_Edit:_ Looking a similar quirks in RestrictedItemList, it appears that it generates clickable links for _all_ methods, but never for getters/setters.  Pretty sure that's a bug in Typedoc then.

Not worth trying to do anything about it for this particular PR, though...